src/helpers/paginate.ts

```diff
@@ -1,12 +1,12 @@
 export const perPage = (requestQuery: {
-  perPage: string,
-  currentPage: string
+  per_page: string,
+  current_page: string
 }) => {
-  const perPageNumber = Number(requestQuery.perPage)
-  const currentPageNumber = Number(requestQuery.currentPage)
+  const perPageNumber = Number(requestQuery.per_page)
+  const currentPageNumber = Number(requestQuery.current_page)
 
-  const perPage = requestQuery.perPage && !isNaN(perPageNumber) ? perPageNumber : 20
-  const currentPage = requestQuery.currentPage && !isNaN(currentPageNumber) ? currentPageNumber : 1
+  const perPage = requestQuery.per_page && !isNaN(perPageNumber) ? perPageNumber : 20
+  const currentPage = requestQuery.current_page && !isNaN(currentPageNumber) ? currentPageNumber : 1
```

Comment:
is it necessary to check `request Query.current page`? if you have put in to currentPage variable.

Reply:
yes, tested on local success

Reply:
hmm okay
i just wondering maybe u just need to check only `currentPageNumber`, so its just:
`const currentPage = !isNaN(currentPageNumber) ? currentPageNumber : 1`

Reply:
yes, that's why we added check `isNaN`

Reply:
updated @khihadysucahyo  e785107